pkgs/development/tools/jo/default.nix

```diff
@@ -0,0 +1,27 @@
+{stdenv, fetchFromGitHub, autoreconfHook}:
+
+stdenv.mkDerivation rec {
+  name="${pname}-${version}";
+  pname="jo";
+  version="0.9";
+
+  src = fetchFromGitHub {
+    owner = "jpmens";
+    repo = "jo";
+
+    rev = "v${version}";
+    sha256="0hv1spbbaca0bvddn0wxk06mm95xnx4zfi99q79hxj1grxvpj1np";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [autoreconfHook];
```

Comment:
If you are already on it you might want add spaces around list entries as well `[ autoreconfHook ]` to better fit our coding style.